scripts/compose.py

```diff
@@ -653,14 +653,25 @@ def _content(self):
 
         if self.build:
             build_spec_parts = self.build.split("@", 1)
+            commit_spec_parts = self.build.split("#", 1)
```

Comment:
if you use the same approach for both cases in the DOcker image this is not needed